VolleyManagement.UnitTests/Mvc/Controllers/GameResultsControllerTests.cs

```diff
@@ -395,9 +398,41 @@ private GameResultsController BuildSUT()
                 _authServiceMock.Object);
         }
 
-        private void VerifyEditGameResult(Times times)
+        private static GameResultViewModel CreateResultViewModel()
         {
-            _gameServiceMock.Verify(ts => ts.EditGameResult(It.IsAny<Game>()), times);
+            var result = new GameResultViewModelBuilder()
+                            .WithTechnicalDefeat(true)
+                            .WithPenalty(new Penalty
+                            {
+                                Amount = 1,
+                                Description = "asd",
```

Comment:
Description to constant?

Reply:
Not sure about that. It won't give too much of a value in scope of the test.